recipes/texture-atlas/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: texture-atlas
+  version: {{ environ['GIT_DESCRIBE_TAG'] ~ "." ~ environ['GIT_DESCRIBE_NUMBER'] ~ "." ~ environ['GIT_DESCRIBE_HASH'] }}
```

Comment:
When the git repository has no tags, rendering the template fails:
`'dict object' has no attribute 'GIT_DESCRIBE_TAG'`.

The error can be bypassed by providing a default like `environ.get('GIT_DESCRIBE_TAG', '')`, etc., but then the version number isn't valid.

What is a good strategy here? Hard-code a version? Create an arbitrary tag on the repo?

Reply:
This is what I did - I created a new tag v1.0.0. I like the fact that the sha and the number of commits since the last tag is included in the file name and I don't know how to do it in a different way.

Reply:
That makes sense, but I don't see any tags in the main repo. Did you only tag your repo?

Reply:
I would have expected the merge to do it, but I guess it does not. So I pushed another tag.